src/subdomains/supporting/payment/services/transaction-helper.ts

```diff
@@ -811,6 +812,9 @@ export class TransactionHelper implements OnModuleInit {
     const isSell = isAsset(from) && isFiat(to);
     const isSwap = isAsset(from) && isAsset(to);
 
+    if (user?.wallet?.amlRuleList?.includes(AmlRule.RULE_SKIP_AML_CHECK) && Config.environment !== Environment.PRD)
+      return;
```

Comment:
wallet and amlRuleList cannot be undefined (`user?.wallet.amlRuleList.includes(...)` should be ok)